controller/resourceStore.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/coreos/etcd/client"
 	"github.com/satori/go.uuid"
 	"golang.org/x/net/context"
+	"github.com/fission/fission"
```

Comment:
nit: mind adding a blank line above this import? (we separate out in-project imports from the rest)